dev/io.openliberty.webcontainer.servlet.6.1.internal.factories/bnd.bnd

```diff
@@ -0,0 +1,107 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 17
+javac.target: 17
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=17))"
+
+Bundle-Name: WAS WebContainer
+Bundle-SymbolicName: io.openliberty.webcontainer.servlet.6.1.internal.factories
+Bundle-Description: WAS WebContainer with object factories for Servlet 6.1 support
+WLP-ServerName: IBM WebSphere Application Server
+WLP-ServerVersion: ${liberty.beta.version}
```

Comment:
Can you open an issue for these two WLP-Server* directives? I only see these in some WebTier bundles so I'd like to clean them up if they are not necessary any longer. Just associate the issue with the Servlet Epic.

Reply:
Checked with Jared.  These 2 can be removed.